src/test/java/org/folio/rest/JsonStoreTest.java

```diff
@@ -71,17 +71,19 @@ public void setup() throws Exception {
   @Test
   public void test1() {
     assertNotNull(jsonStore.getCollection(null, null, null));
-    assertTrue(jsonStore.getCollection(null, null, null).size() == 10);
-    assertTrue(jsonStore.getCollection(null, null, null).get(0).getString("name").equals("thing1"));
+    assertTrue(jsonStore.getCollection(null, null, null).getObjectList().size() == 10);
+    assertTrue(jsonStore.getCollection(null, null, null).getTotalRecords() == 10);
+    assertTrue(jsonStore.getCollection(null, null, null).getObjectList().get(0).getString("name").equals("thing1"));
```

Comment:
SonarCloud complains
https://sonarcloud.io/project/issues?id=org.folio%3Amod-users-bl&pullRequest=116&resolved=false&types=CODE_SMELL
that assertTrue(... == ...) and assertTrue(...equals(...))
should be replaced.
You may use hamcreast that rest-assured ships with:
assertThat(..., is(...))